### DISCUSSION TITLE
Milvus Cluster

### DISCUSSION THREAD

**Discussion by engrabdu** [milvus-io/milvus#19105] [Q&A and General discussion] [answered]
Hello all
How to check the Milvus Cluster Status with other nodes that milvus mishared is running or not ..can any one explain  ?
thanks

**Comment by yhmo** [comment, marked as answer]
Mishard is a middleware for milvus v1.x. It manages a write-node and several read-nodes to construct a basic distribution service. But it has no ability to monitor the state of the nodes. Each node is started by a Milvus docker image, we can use `docker logs` to check the state of a node.
